var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -42,6 +42,7 @@ class Hdf5(Package):
     version('1.8.13', 'c03426e9e77d7766944654280b467289')
 
     variant('debug', default=False, description='Builds a debug version of the library')
+    variant('shared', default=False, description='Builds a static executable version of the library')
```

Comment:
Hi. I think the variant `shared` was not present in this particular package as hdf5 builds by default both static and shared libraries. If there's the need to disable the build of shared libraries, would it be a problem to keep `default=True` to be more consistent with other packages?